include/zephyr/bluetooth/gatt.h

```diff
@@ -1772,9 +1787,30 @@ struct bt_gatt_read_params {
 			bool variable;
 		} multiple;
 		struct {
-			/** First requested handle number. */
+			/** First requested attribute handle number.
+			 *
+			 * @note The starting handle is set to the starting
+			 * point of the range over which this read should
+			 * be performed. For example, this could be set to
+			 * @ref BT_ATT_FIRST_ATTRIBUTE_HANDLE to set the
+			 * starting point of the range at the beginning of
+			 * the GATT database.
+			 *
+			 * @note This value is also automatically
+			 * incremented by the stack after processing each
+			 * matching handle-value pair returned by the
+			 * server.
+			 */
 			uint16_t start_handle;
-			/** Last requested handle number. */
+			/** Last requested attribute handle number.
+			 *
+			 * @note The ending handle is set to the ending
+			 * point of the range over which this read should
+			 * be performed. For example, this could be set to
+			 * @ref BT_ATT_LAST_ATTRIBUTE_HANDLE to set the
+			 * ending point of the range at the end of the GATT
+			 * database.
+			 */
```

Comment:
Or to the end handle service after discovery where the end_handle is available in the `bt_gatt_service_val`.